pkg/iostreams/iostreams.go

```diff
@@ -406,13 +406,16 @@ func System() *IOStreams {
 		term:         &terminal,
 	}
 
-	stdoutIsTTY := io.IsStdoutTTY()
+	io.stdoutIsTTY = io.IsStdoutTTY()
+	io.stderrIsTTY = io.IsStderrTTY()
+	io.stdoutTTYOverride = true
+	io.stderrTTYOverride = true
```

Comment:
Now that `stdout` and `stderr` TTY statuses have been determined set the override so we do not do the actual check again, but instead use the values found above.

Reply:
I'd say that this is a premature optimization that we should avoid unless we have a good reason to perform. The override mechanism should ideally only be used for testing. Let's assume that `io.IsStdoutTTY()` is cheap and that we can safely repeat it in RefreshScreen